python/pyspark/errors/exceptions/connect.py

```diff
@@ -119,13 +135,43 @@ class QueryExecutionException(SparkConnectGrpcException, BaseQueryExecutionExcep
     """
 
 
-class SparkUpgradeException(SparkConnectGrpcException, BaseSparkUpgradeException):
+class PythonException(SparkConnectGrpcException, BasePythonException):
     """
-    Exception thrown because of Spark upgrade from Spark Connect
+    Exceptions thrown from Spark Connect server.
```

Comment:
qq: Is `Spark Connect server` and `Spark Connect` different??

Only `PythonException` says it's thrown from Spark Connect "server".

Reply:
The comment is from the previous. We can change it to `Spark Connect` while we are here.